source/common-tasks/os.markdown

```diff
@@ -48,6 +48,27 @@ Updates of the {% term "Home Assistant Operating System" %} are independent of o
 
 {% endtabbed_block %}
 
+#### About boot slots used during the update
+
+The {% term "Home Assistant Operating System" %} uses two boot slots. On first installation, boot slot A is used. After that, for each update, the other boot slot is updated. On reboot, the system boots from the other boot slot (A ➝ B ➝ A,...). When booting fails, the system automatically uses the other boot slot, so that it boots from the last working operating system.
```

Comment:
As it stands, it might be understood that any reboot would switch the boot slot. Maybe like so?

```suggestion
The {% term "Home Assistant Operating System" %} uses two boot slots. On first installation, boot slot A is used. After that, on each Operating System update, the other boot slot is updated and reboot is triggered. On that reboot, the system boots from the other boot slot (A ➝ B ➝ A,...). When booting fails, the system automatically uses the previous boot slot, so that it boots from the last working operating system.
```